Makefile

```diff
@@ -70,21 +60,6 @@ push:
 pull:
 	docker pull $(IMG)
 
-run:
```

Comment:
I missed this `make run` to playaround with the kernel image environment. Also this is documented in README.

But readme is really outdated it says Linux 5.5.x, RV64IMA, Ubuntu, it deserves an update.

Reply:
Right, there are some details to iron out, such as the `run` you mentioned. `run-as-root` may work regardless. This was being setup in cartesi/toolchain previously.

I'm currently thinking of migrating most of it to the sdk repo.
But I have to thing about the details to avoid circular dependencies.